packages/web/src/common/store/social/collections/sagas.ts

```diff
@@ -62,8 +63,9 @@ export function* repostCollectionAsync(
   action: ReturnType<typeof socialActions.repostCollection>
 ) {
   yield* call(waitForWrite)
-  const userId = yield* select(getUserId)
-  const isGuest = yield* select(getIsGuestAccount)
+  const accountUser = yield* queryAccountUser()
+  const { user_id: userId } = accountUser ?? {}
+  const isGuest = yield* call(selectIsGuestAccount, accountUser)
```

Comment:
do we want these selectors or just simply descructure accountUser or do the selectors have a bit of logic?